libr/core/cmd_debug.c

```diff
@@ -5238,7 +5238,11 @@ static int cmd_debug(void *data, const char *input) {
 					}
 				}
 				if (stop != -1) {
-					P ("stopreason=%s\n", r_debug_reason_to_string (stop));
+					if (strcmp (r_debug_reason_to_string (core->dbg->reason.type), "signal") == 0) {
```

Comment:
why convert to string and use == 0 when you just need to core->dbg->reason.type == R_DEBUG_REASON_SIGNAL ?